SWIPs/swip-message_to_honey.md

```diff
@@ -13,28 +13,33 @@ Created: 31-07-2019
 
 ## Specification
 <!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for the current Swarm platform and future client implementations.-->
-* The Swarm source-code references the address of a smart-contract which acts as a honey pricing oracle for messages.
-* The price oracle implements the `MsgToHoneyPrice` interface (to-be specified) 
-* The `MsgToHoneyPrice` contract will be initially owned by an M/N multi-sig of Swarm developers and stakeholders.
-* Upon receiving/sending of a message, a node checks his local database if his cached price is valid. If not, the node will query the price oracle to get the current price. The node will apply the current price to the balance of his peer. (see diagram below)
+* The Swarm source code references the address of a smart-contract which acts as a honey pricing oracle for messages.
+* The price oracle implements the `MsgToHoney` interface (to-be specified) 
+* The `MsgToHoney` contract will be initially owned by an M/N multi-sig of Swarm developers and stakeholders.
+* Upon receiving/sending of a message, a node checks its local database if its cached price is valid. If not, the node will query the price oracle to get the current price. The node will apply the current price to the balance of its peer (see diagram below).
 * Nodes expect other nodes to apply the same price as they do themselves. Due to the asynchronicity of the network, this will not necessarily be true around the period that prices are updated, which causes accounting dust (pollen) to be collected. This SWIP does not facilitate a solution for this, as the expected amount of pollen is not high enough to cause disruptions in the network. 
```

Comment:
I don't agree to this pollen language (I think I commented the same elsewhere).

Also, it is an actual conflict with what you wrote earlier: that it is atomic. If it is atomic, then there should be no difference.